manifests/config.pp

```diff
@@ -1,5 +1,6 @@
 # Configuration for Candlepin
 class candlepin::config {
+  assert_private()
```

Comment:
What does this do at runtime?

Reply:
https://github.com/puppetlabs/puppetlabs-stdlib#assert_private is not that clear, but essentially if you `include candlepin::config` from another module (such as `katello`) it'll raise an error. I'm not entirely sure about the second commit. Maybe just marking it as `@api private` in the docs is enough (https://github.com/theforeman/puppet-candlepin/pull/117) but this way we can easily revert it.